composer/blog/gcp-tech-blog/data-orchestration-with-composer/requirements.txt

```diff
@@ -0,0 +1 @@
+apache-airflow[gcp]==1.10.14
```

Comment:
do you need to include the backport providers? (I know they're installed in Composer, but we have to manually specify them here)

Reply:
ahh we don't actually, I'll remove it. Is this file still required (it will be empty)?